transformer_engine/common/include/transformer_engine/multi_tensor.h

```diff
@@ -17,13 +17,58 @@
 extern "C" {
 #endif
 
+/*!  \brief Computes L2 norm for a list of contiguous tensors, providing the option to
+ *          immediately exit the kernel based on the value of the 'noop' tensor.
+ *
+ * \warning   This API is **experimental** and subject to change.
+ * \warning   Argument device_id is deprecated and will be removed in a future release.
+ *
+ *  \param[in]     chunk_size              Number of tensors per kernel launch,
+                                           limited by total size of kernel args.
```

Comment:
The description I gave you earlier wasn't correct. Here is a better description:
```suggestion
 *  \param[in]     chunk_size              Number of tensor elements processed by a CUDA block.
```